### ISSUE TITLE
LevelDB query files named by concatenating table fields in RiakTS 1.5 [JIRA: RTS-1760]

### ISSUE THREAD

**Issue by nerophon** [basho/riak_kv#1625]
https://github.com/basho/riak_kv/blob/riak_ts-develop/src/riak_kv_qry_buffers_ldb.erl#L43

The above code causes queries to fail where the table has a lot of fields, because the filename exceeds the limit permitted by the OS. This was experienced by a customer; the following log was retrieved:

```
2017-02-07 07:49:24.865 [warning] <0.604.0>@riak_kv_qry_buffers_ldb:new_table:60 qbuf eleveldb:open(/mnt/riakdata/data/query_buffers/test_transactions_13_id+bedeplayerid+remoteref+sitecode+correlationtoken+transactiondate+category+banking_type+banking_subtype+banking_remoteref+banking_currency+banking_amount+banking_description+banking_providername+banking_paymententitydetails+gaming_type+gaming_gamename+gaming_gamecode+gaming_gamingeventid+gaming_channel+gaming_currency+gaming_actionid+gaming_actionstatus+gaming_actionamount+gaming_actiondescription+sports_type+sports_sportseventid+sports_currency+sports_amount+loyalty_type+loyalty_points+loyalty_pointsbrand+loyalty_description+bonus_type+bonus_credit+bonus_debit+bonus_bonusname+bonus_promocode+bonus_currency_transactiondate.al__00004537640000848446) failed: {db_open,"IO error: /mnt/riakdata/data/query_buffers/test_transactions_13_id+bedeplayerid+remoteref+sitecode+correlationtoken+transactiondate+category+banking_type+banking_subtype+banking_remoteref+banking_currency+banking_amount+banking_description+banking_providername+banking_paymententitydetails+gaming_type+gaming_gamename+gaming_gamecode+gaming_gamingeventid+gaming_channel+gaming_currency+gaming_actionid+gaming_actionstatus+gaming_actionamount+gaming_actiondescription+sports_type+sports_sportseventid+sports_currency+sports_amount+loyalty_type+loyalty_points+loyalty_pointsbrand+loyalty_description+bonus_type+bonus_credit+bonus_debit+bonus_bonusname+bonus_promocode+bonus_currency_transactiondate.al__00004537640000848446/LOCK: File name too long"}
2017-02-07 07:49:24.866 [warning] <0.8015.0>@riak_kv_qry:maybe_create_query_buffer:310 Failed to set up query buffer for {riak_select_v3,{riak_sel_clause_v1,rows,[],[],[],[{identifier,[<<"*">>]}],[]},<<"test_transactions_13">>,[{and_,{'<=',<<"transactiondate">>,{integer,1485295200000}},{and_,{'=',<<"bedeplayerid">>,{integer,818405548142760034}},{'>=',<<"transactiondate">>,{integer,1485122400000}}}}],[{<<"transactiondate">>,asc,nulls_last}],"\n",riak_ql_table_test_transactions_13_320523031941664944417524937300681317050,none,false,sql,undefined,undefined,[],[1],undefined}: db_open
```

This bug will automatically be fixed when https://github.com/basho/riak_kv/pull/1601 is completed. However, it may be appropriate to produce a patch in the meantime.

**Comment by Basho-JIRA** [comment]
The customer has expressed a desire to get these new 1.5 features into their next sprint which concludes Feb 20th. So that would be a patch or minor version if we're going to keep them happy there.
* Zendesk ticket is here: https://basho.zendesk.com/agent/tickets/14384 

***[posted via JIRA by Derek Somogyi]***

**Comment by Basho-JIRA** [comment]
Fix pushed to https://github.com/basho/riak_kv/pull/1626. 

***[posted via JIRA by Andrei Zavada]***

**Comment by hmmr** [comment]
Fix merged to riak_ts-develop-1.5, will appear in 1.5.2.

**Comment by Basho-JIRA** [comment]
Fix merged. 

***[posted via JIRA by Andrei Zavada]***
